cmd/podman/machine/set.go

```diff
@@ -34,7 +45,33 @@ func init() {
 	flags := setCmd.Flags()
 
 	rootfulFlagName := "rootful"
-	flags.BoolVar(&setOpts.Rootful, rootfulFlagName, false, "Whether this machine should prefer rootful container execution")
+	flags.BoolVar(&setFlags.Rootful, rootfulFlagName, false, "Whether this machine should prefer rootful container execution")
```

Comment:
I too think `rootfull` is more correct, but I  that ship has all but sailed.  We've already established `rootful` in a number of places, including RHEL docs.

Reply:
The man pages have both, I will open a PR to fix this and alias the --rootful and --rootfull options in podman machine.
Lets still to rootfull since it line up well with rootless.

Reply:
https://github.com/containers/podman/pull/13964

Reply:
https://github.com/containers/podman/pull/13964#issuecomment-1106546414

Reply:
I'd also be in favor of dropping the suffix entirely, and changing the setting to --root, or --connection=root|rootless

Reply:
I would be fine with switching it to --root, but `podman --root` has this option.

Switching the option to a param is not a great solution since the option already exists.